geo-data-server/src/main/java/org/gridsuite/geodata/server/GeoDataService.java

```diff
@@ -37,9 +41,11 @@ public class GeoDataService {
 
     private static final Logger LOGGER = LoggerFactory.getLogger(GeoDataService.class);
 
+    private DefaultSubstationGeoDataByCountry defaultSubstationsGeoData;
```

Comment:
You could use the Spring way for this: @Service and injection

Reply:
Done